en/docs/integrations/integration-concepts.md

```diff
@@ -8,18 +8,18 @@ Integration in Choreo is the process of connecting APIs, events, and streams to
 An integration template in Choreo is a prebuilt sample designed to address a common integration requirement. Choreo provides a comprehensive set of integration templates for you to select and use depending on your requirement.
 
 ## Trigger
-A trigger in Choreo is an event or an action that can cause a Choreo application to start executing.
+A trigger in Choreo is an event or an action that can cause an integration to start executing.
 
 Choreo supports the following trigger types:
 
- - **Schedule**: To trigger an application based on a schedule that you set.
- - **Manual**: To run and test an application manually.
- - **Calendar**: To trigger an application based on events in a Google Calendar.
- - **GitHub**: To trigger an application based on GitHub events and actions.
- - **Salesforce**: To trigger an application based on topics in a Salesforce account. 
+ - **Schedule**: To trigger an integration based on a schedule that you set.
+ - **Manual**: To run and test an integration manually.
+ - **Calendar**: To trigger an integration based on events in a Google Calendar.
+ - **GitHub**: To trigger an integration based on GitHub events and actions.
+ - **Salesforce**: To trigger an integration based on topics in a Salesforce account. 
 
 ## API Call
-An API call in Choreo allows you to connect to and interact with an external service through your application. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
+An API call in Choreo allows you to connect to and interact with an external service through your integration. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
```

Comment:
we need to make this generic right? cz we have API call in services as well..

Reply:
Changed.